app/migrations/seed.py

```diff
@@ -388,6 +390,15 @@ def create_float_transfer_account(app):
             db.session.commit()
     print_section_conclusion('Done Creating/Updating Float Wallet')
 
+def make_rejected_transactions_unstarted(app):
+    print_section_title('Marking rejected transactions as unstarted')
+    rejected_transactions = db.session.query(CreditTransfer)\
+        .filter(CreditTransfer.transfer_status == TransferStatusEnum.REJECTED)\
+        .execution_options(show_all=True)
+    for rt in rejected_transactions:
+        rt.blockchain_status = BlockchainStatus.UNSTARTED
+    db.session.commit()
```

Comment:
@michielderoos can we please have this as a raw SQL operation directly in the `bb92df9cb6e3_.py` migration. This means the data is changed only once when DB is migrated and in a sequential order that is expected. As `seed.py` is run basically every time the app is spun up or updated

Reply:
I also spoke to @enjeyw about this and he concurs